src/index.js

```diff
@@ -1,71 +1,19 @@
 import './style.css';
-// To-do list Array of Objects
-const todolist = [
-  {
-    index: 0,
-    description: 'Wash the Car',
-    completed: false,
-  },
-  {
-    index: 1,
-    description: 'Clean the Garden',
-    completed: false,
-  },
-  {
-    index: 2,
-    description: 'Study Javascript Tutorials',
-    completed: false,
-  },
-];
-class Tasks {
-  constructor() {
-    this.toDoList = [];
-  }
+import TodoList from './todolist';
 
-  add(description, completed) {
-    this.toDoList = this.toDoList.concat({
-      index: Date.now(),
-      description,
-      completed,
-    });
-  }
+const todoList = new TodoList();
+todoList.displayTasks();
 
-  remove(id) {
-    this.toDoList = this.toDoList.filter((list) => list.id !== Number(id));
-  }
-}
-// new task
-const lists = new Tasks();
-const todoDiv = document.querySelector('.lists');
-const getTodoList = () => {
-  todolist.forEach((list) => {
-    const li = document.createElement('li');
-    li.classList.add('list');
-    const liDiv = document.createElement('div');
-    liDiv.classList.add('li-div');
-    // checkbox
-    const checkbox = document.createElement('input');
-    checkbox.type = 'checkbox';
-    checkbox.checked = list.completed;
-    liDiv.appendChild(checkbox);
-    // description
-    const desc = document.createElement('p');
-    desc.innerText = list.description;
-    liDiv.appendChild(desc);
-    li.appendChild(liDiv);
-    // vertical dots
-    const dots = document.createElement('i');
-    dots.classList.add('fa');
-    dots.classList.add('fa-ellipsis-v');
-    li.appendChild(dots);
-    todoDiv.appendChild(li);
-  });
-};
-getTodoList();
-// Add function
-const listInput = document.getElementById('todo-input');
-listInput.addEventListener('keydown', (e) => {
-  if (e.key === 'Enter') {
-    lists.setTodo();
-  }
+const form = document.querySelector('form');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const newActivity = form.elements.new_task.value;
+  todoList.addTask(newActivity);
+  todoList.displayTasks();
+  form.reset();
```

Comment:
- [ ]  I am trying to create a task so that I can check if you've implemented other functionality correctly, but I can't because every time I submit a task, the page immediately reload. and I can't see the list of those tasks. fix that please and make sure that all these requirements are implemented correctly.
![image](https://user-images.githubusercontent.com/43877689/173564098-6c811096-e9d5-48af-916a-c084e18001f6.png)

Reply:
@ishimwezachee  Thank you for your time in reviewing this work. However I don't understand what you are asking me to implement as I have gone through the project requirements and I think I have implemented all that is required at this stage.
Thanks once again for your time.